src/devices/schneider_electric.ts

```diff
@@ -311,6 +317,83 @@ const schneiderElectricExtend = {
             ],
         };
     },
+
+    dimmingMode: (): ModernExtend => {
+        const extend = enumLookup({
+            name: 'dimmer_mode',
+            lookup: {
+                Auto: 0,
+                'RL-LED': 3,
+            },
+            cluster: 'lightingBallastCfg',
+            attribute: 'wiserControlMode',
+            description: 'Controls the dimmer mode.',
+            entityCategory: 'config',
+        });
+        extend.configure.push(setupConfigureForReading('lightingBallastCfg', ['wiserControlMode']));
+        return extend;
+    },
+
+    occupancyConfiguration: (): ModernExtend => {
+        const extend = enumLookup({
+            name: 'occupancy_sensitivity',
+            lookup: {
+                Low: 50,
+                Medium: 75,
+                High: 100,
+            },
+            cluster: 'msOccupancySensing',
+            attribute: 'schneiderOccupancySensitivity',
+            description: 'Sensitivity of the occupancy sensor',
+            entityCategory: 'config',
+        });
+
+        const luxScale: ScaleFunction = (value: number, type: 'from' | 'to') => {
+            if (type === 'from') {
+                return Math.round(Math.pow(10, (value - 1) / 10000));
+            } else {
+                return Math.round(10000 * Math.log10(value) + 1);
+            }
+        };
+
+        const luxThresholdExtend = numeric({
+            name: 'ambience_light_threshold',
+            cluster: 'manuSpecificSchneiderOccupancyConfiguration',
+            attribute: 'ambienceLightThreshold',
+            reporting: {min: '10_SECONDS', max: '1_HOUR', change: 5},
+            description: 'Threshold above which occupancy will not trigger the light switch.',
+            unit: 'lx',
+            scale: luxScale,
+            entityCategory: 'config',
+            valueMin: 1,
+            valueMax: 1000,
+        });
+        extend.fromZigbee.push(...luxThresholdExtend.fromZigbee);
+        extend.toZigbee.push(...luxThresholdExtend.toZigbee);
+        extend.exposes.push(...luxThresholdExtend.exposes);
+        extend.configure.push(
+            setupConfigureForReading('manuSpecificSchneiderOccupancyConfiguration', ['ambienceLightThreshold']),
+            setupConfigureForReading('msOccupancySensing', ['schneiderOccupancySensitivity']),
+        );
+
+        return extend;
+    },
+
+    dimmingOnLevel: (): ModernExtend => {
```

Comment:
Schneider dimmers don't support fixed on_levels for some reason, just 100% or previous.

Reply:
FYI: `on_level` works with arbitrary levels for NHMOTION/UNIDIM/1 (when set via Dev console), so I don't think this is true in general.

Reply:
Thanks, you are very much correct. I guess I assumed it didn't work based on the app not supporting it.

I guess we could revert to the default levelControl implementation, but as you pointed out in your PR the UI for it seems broken. It looks like numeric expose with presets are broken. I'll poke at it a bit.